src/main/java/controller/BlackjackController.java

```diff
@@ -0,0 +1,44 @@
+package controller;
+
+import domain.PlayerIntentionType;
+import domain.card.Deck;
+import domain.user.Dealer;
+import domain.user.Player;
+import domain.user.Players;
+import service.BlackjackService;
+
+import static view.InputView.*;
+import static view.OutputView.*;
+
+public class BlackjackController {
+	public static void run(Deck deck, Dealer dealer, Players players) {
+		BlackjackService.giveInitialCards(deck, dealer, players);
+		printInitialStatus(dealer.openCard(), players);
+
+		if (dealer.isNotBlackjack()) {
+			players.forEach(player -> proceedExtraDraw(player, deck));
+			while (dealer.canDrawMore()) {
+				BlackjackService.addCard(dealer, deck);
+				printDealerDrawing();
+			}
+		}
+
+		printResult(dealer, players);
+	}
+
+	private static void printResult(Dealer dealer, Players players) {
+		printResultStatus(dealer.openAllCards(), players);
+		printTotalResult(BlackjackService.createResult(dealer, players), players);
+	}
+
+	private static void proceedExtraDraw(Player player, Deck deck) {
+		while (player.canDrawMore() && wantDraw(player)) {
+			BlackjackService.addCard(player, deck);
+			printCardsStatusOf(player);
+		}
+	}
+
+	private static boolean wantDraw(Player player) {
+		return PlayerIntentionType.YES.equals(PlayerIntentionType.of(inputPlayerIntention(player)));
```

Comment:
PlayerIntentionType값을 생성해서 equals하는 것 보단 PlayerIntentionType에게 "y"가 맞는지 물어보면 어떨까요?
```suggestion
		return PlayerIntentionType.isYes(inputPlayerIntention(player));
```

Reply:
피드백을 받고 헉 했습니다..!

Enum을 객체라고 인지하고 있었지만 이부분까지 메시지를 보내야한다고 생각을 못했네요..!

확실히 .equals를 사용하는 것보다 원하는 행위에 대한 메서드만 열어주는 편이 낫고 가독성도 뛰어나군요👍

의견주신 부분 반영해서 커밋했습니다.

**Enum에도 메시지를 보내라..! 잘 기억하겠습니다.**

감사합니다😀